unity-renderer/Assets/DCLServices/PlacesAPIService/PlacesAPIClient.cs

```diff
@@ -0,0 +1,120 @@
+
+            return response;
+        }
+
+        public async UniTask<IHotScenesController.PlaceInfo> GetPlace(Vector2Int coords, CancellationToken ct)
+        {
+            const string URL = BASE_URL + "?positions={0},{1}&with_realms_detail=true";
+            var result = await webRequestController.GetAsync(string.Format(URL, coords.x, coords.y), cancellationToken: ct);
+
+            if (result.result != UnityWebRequest.Result.Success)
+                throw new Exception($"Error fetching place info:\n{result.error}");
+
+            var response = Utils.SafeFromJson<IHotScenesController.PlacesAPIResponse>(result.downloadHandler.text);
+
+            if (response == null)
+                throw new Exception($"Error parsing place info:\n{result.downloadHandler.text}");
+
+            if (response.data.Count == 0)
+                throw new NotAPlaceException(coords);
+
+            return response.data[0];
+        }
+
+        public async UniTask<IHotScenesController.PlaceInfo> GetPlace(string placeUUID, CancellationToken ct)
+        {
+            var url = $"{BASE_URL}/{placeUUID}?with_realms_detail=true";
+            var result = await webRequestController.GetAsync(url, cancellationToken: ct);
+
+            if (result.result != UnityWebRequest.Result.Success)
+                throw new Exception($"Error fetching place info:\n{result.error}");
+
+            var response = Utils.SafeFromJson<IHotScenesController.PlacesAPIGetParcelResponse>(result.downloadHandler.text);
+
+            if (response == null)
+                throw new Exception($"Error parsing place info:\n{result.downloadHandler.text}");
+
+            if (response.ok == false)
+                throw new NotAPlaceException(placeUUID);
+
+            if (response.data == null)
+                throw new Exception($"No place info retrieved:\n{result.downloadHandler.text}");
+
+            return response.data;
+        }
+
+        public async UniTask<List<IHotScenesController.PlaceInfo>> GetFavorites(CancellationToken ct)
+        {
+            const string URL = BASE_URL + "?only_favorites=true&with_realms_detail=true";
+            UnityWebRequest result = await webRequestController.GetAsync(URL, isSigned: true, cancellationToken: ct);
+            var response = Utils.SafeFromJson<IHotScenesController.PlacesAPIResponse>(result.downloadHandler.text);
+
+            if (response == null)
+                throw new Exception($"Error parsing get favorites response:\n{result.downloadHandler.text}");
+
+            if (response.data == null)
+                throw new Exception($"No favorites info retrieved:\n{result.downloadHandler.text}");
+
+            return response.data;
+        }
+
+        public async UniTask SetPlaceFavorite(string placeUUID, bool isFavorite, CancellationToken ct)
+        {
+            const string URL = BASE_URL + "/{0}/favorites";
+            string payload = "{\"favorites\":" + isFavorite.ToString().ToLower() + "}";
```

Comment:
Not a big deal but it looks like you can construct the payload in a `const` manner: for true and false to avoid allocations

Reply:
A bit boilerplaity but love the idea, thanks!